servicetalk-http-api/src/main/java/io/servicetalk/http/api/BlockingHttpService.java

```diff
@@ -17,15 +17,14 @@
 
 import io.servicetalk.concurrent.GracefulAutoCloseable;
 import io.servicetalk.concurrent.api.Single;
-import io.servicetalk.transport.api.ExecutionStrategyInfluencer;
 
 import static io.servicetalk.http.api.DefaultHttpExecutionStrategy.OFFLOAD_RECEIVE_DATA_STRATEGY;
 
 /**
  * The equivalent of {@link HttpService} but with synchronous/blocking APIs instead of asynchronous APIs.
  */
 @FunctionalInterface
-public interface BlockingHttpService extends ExecutionStrategyInfluencer<HttpExecutionStrategy>, GracefulAutoCloseable {
+public interface BlockingHttpService extends HttpExecutionStrategyInfluencer, GracefulAutoCloseable {
```

Comment:
Should services implement influencer? They don't in 0.41. If a different strategy is necessary, users can use a router to specify one. 
On the other hand, better to keep a definition of the strategy close to the implementation of the service.

Reply:
In 0.41 and earlier individual instances of service could implement `HttpExecutionStrategyInfluencer` and it would influence the execution strategy used. 0.42 just makes the need to implement the interface a requirement and adds a default for each API model. The intention was that if something can be an influencer then the parent interface should be explicit about that.

Reply:
Thanks for clarifying, sounds good!